Source/Core/Core/HW/WiimoteEmu/WiimoteEmu.cpp

```diff
@@ -780,7 +777,7 @@ bool Wiimote::IsUpright() const
 void Wiimote::SetRumble(bool on)
 {
   const auto lock = GetStateLock();
-  m_motor->control_ref->State(on);
+  m_rumble->controls.front()->control_ref->State(on);
```

Comment:
Since this is a collection, should it potentially enable all of them (vs. assuming there is just one in there)?

Reply:
This particular collection always just has one element. I realize this is somewhat ugly but the few outputs are a bit of a special case compared to the inputs.